src/phone_sms_terms.js

```diff
@@ -70,7 +71,12 @@ var mobilePhoneUpdate = function() {
       $('#id_sms_subscribed').prop('checked', true);
       $('#id_sms_subscribed').val('sms_subscribed');
     }
-    $(SMS_SUBSCRIBE_DIV).show(); //todo: css for slide down
+    if ((/always-hide/).test(phonemobile)) {
+      $('#id_sms_subscribed').prop('checked', false);
```

Comment:
Do we also need to set the value of `#id_sms_subscribed` to `''`? I can take a look and do some testing.

Reply:
I don't think it should matter, since it's unchecked, but I'd appreciate if you could take a look!

Reply:
You're completely right. We actually ran into this when pairing last week, but I had already forgotten. "If a checkbox is unchecked when its form is submitted, neither the name nor the value is submitted to the server." So, even if the value somehow were to get set for `#id_sms_subscribed`, there's no problem, because it's unchecked here and not exposed in the UI to be checked. Testing confirms this. Thanks for giving me some time to think on this one!